mlvm/mlvm.sh

```diff
@@ -218,9 +226,6 @@ function main() {
   # Install Pip packages
   echo "Installing Pip Packages"
   install_pip_packages
-
-  # Clean up environment
-  conda clean -y --all
```

Comment:
Why this is not necessary?

Reply:
I moved it here as cleaning up with Mamba is much faster.

https://github.com/GoogleCloudDataproc/initialization-actions/blob/94305303a69eec86a5af395b627cf8d5b0d7d831/mlvm/mlvm.sh#L148-L149